app/Helpers/render/site-award.php

```diff
@@ -140,7 +140,8 @@ function RenderAwardGroup(array $awards, string $title, string $awardsOwnerUsern
 
     echo "<div id='" . strtolower(str_replace(' ', '', $title)) . "'>";
     echo "<h3 class='flex justify-between gap-2'><span class='grow'>$title</span>$counters</h3>";
-    echo "<div class='component flex flex-wrap justify-start gap-2'>";
+    echo "<div class='flex justify-center'>";
+    echo "<div class='component flex flex-wrap gap-2 justify-start bg-embed w-full xl:rounded xl:py-2 xl:px-5'>";
```

Comment:
We could easily fit 6 per row 🤔 This would reduce the overall height occupied of awards in their current state. Just an idea.

![Screenshot 2023-09-04 at 19 38 29](https://github.com/RetroAchievements/RAWeb/assets/1280590/44e2c013-6320-4ee8-bba3-67703874a4f3)

Reply:
Considered this, but I think users would be pretty upset with the change. Based on everything learned from building drag & drop site awards, users are primarily ordering their awards for desktop with 5 badges per row (even though mobile is variable on # shown per row).

Reply:
Alright. Let's leave it at that for now.

On a side note: the question is which users - we'll get a lot of strong feedback from some on making the whole site wider, too. 
I'd like to modernize this site eventually ;)

Reply:
> the question is which users

I'm one of them, and that's my profile that you've used to illustrate 6 per row.

Note how the Lolo achievements at the end of the list are in the middle of a row (they start a row on my live profile page). And less obviously, the Space Quest icon starts a block of adventure icons leading up to the Lolo achievements:

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e947834f-3478-4f90-b7a6-d4d2c585d551)

Then I've got five mickey mouse icons, then other cartoon-related stuff.

Reply:
I wouldn't say I'd be upset by the change, but my badges are currently ordered under the assumption of 5 per row.

Reply:
fair enough :) it does get very dense with 6 per row, too, so i don't like it too much either after looking at it for longer.
that came off a bit strong i guess, sorry for that.